src/backend/distributed/commands/vacuum.c

```diff
@@ -233,14 +250,13 @@ VacuumTaskList(Oid relationId, int vacuumOptions, List *vacuumColumnList)
  * statements.
  */
 static StringInfo
-DeparseVacuumStmtPrefix(int vacuumFlags)
+DeparseVacuumStmtPrefix(CitusVacuumParams vacuumParams)
 {
+	int vacuumFlags = vacuumParams.options;
 	StringInfo vacuumPrefix = makeStringInfo();
 	const int unsupportedFlags PG_USED_FOR_ASSERTS_ONLY = ~(
+		VACOPT_VACUUM |
 		VACOPT_ANALYZE |
-		VACOPT_DISABLE_PAGE_SKIPPING |
```

Comment:
Why do we end-up with changing this with this commit? Is there anything changing related to `VACOPT_FULL ` or `VACOPT_FREEZE ` ?

Reply:
Later in the code, after the assert, we have handling for those options. Meanwhile we don't handle those options before the assert